daemon/algod/api/server/v2/handlers.go

```diff
@@ -1432,6 +1435,11 @@ func (v2 *Handlers) GetLedgerStateDelta(ctx echo.Context, round uint64, params m
 	if err != nil {
 		return notFound(ctx, err, fmt.Sprintf(errFailedRetrievingStateDelta, err), v2.Log)
 	}
+	if handle == protocol.JSONStrictHandle {
+		// Zero out the Txleases map since it cannot be represented in JSON, as it is a map with an
+		// object key.
```

Comment:
```suggestion
		// Zero out the Txleases map since it cannot be represented in JSON, as it is a map with an
		// object key. The information is readily available from the transaction themselves.
```

Reply:
Good improvement but I don't want to dismiss the two approvals for this (and because flakey tests are particularly bad right now...)